lib/smart_proxy_dynflow_core/log.rb

```diff
@@ -67,20 +110,32 @@ def call(_severity, _datetime, _prog_name, message)
           message
         end
       end
-
-      def format(message)
-        call(nil, nil, nil, message)
-      end
     end
 
     class ProxyAdapter < ::Dynflow::LoggerAdapters::Simple
       def initialize(logger, level = Logger::DEBUG, _formatters = [])
         @logger           = logger
         @logger.level     = level
-        @logger.formatter = ProxyStructuredFormater.new(@logger)
         @action_logger    = apply_formatters(ProgNameWrapper.new(@logger, ' action'), [ProxyStructuredFormater])
         @dynflow_logger   = apply_formatters(ProgNameWrapper.new(@logger, 'dynflow'), [ProxyStructuredFormater])
       end
     end
   end
 end
+
+class PryAction < ::Dynflow::Action
```

Comment:
This should go away before merge

Reply:
Or we could turn it into something like `SmartProxyDynflowCore::DummyAction` with docs comments on how to use it: it might be quite nice debugging setup.

Reply:
Shouldn't this then live directly in Dynflow so it can be used anywhere?